client/src/apis/chessGameApi.js

```diff
@@ -0,0 +1,7 @@
+const SERVER_API_URL = `${process.env.REACT_APP_SERVER_URL}/api`;
```

Comment:
REACT_APP_SERVER_URL -> this has been set in configuration file at config/globalConfig
Please pick it up from there.